setup.py

```diff
@@ -0,0 +1,40 @@
+"""Setup for vectordraw XBlock."""
+
+import os
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, _, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='vectordraw-xblock',
+    version='0.1',
+    description='vectordraw XBlock',   # TODO: write a better description.
+    packages=[
+        'vectordraw',
+    ],
+    install_requires=[
+        'XBlock',
+        'xblock-utils',
```

Comment:
@singingwolfboy Thanks for catching that. The Vector Drawing XBlock itself actually doesn't make use of `pyyaml` (it only depends on it because it extends `XBlock`). So instead of adding it here I removed `pyyaml` from `requirements.txt` and bumped the `XBlock` version to `0.4.2` (which lists `pyyaml` in its [`requirements.txt` file](https://github.com/edx/XBlock/blob/bfa7d968157c188d8e4d80e5ded839248c490ce1/requirements.txt#L2)).